tianshou/policy/modelfree/MAdqn.py

```diff
@@ -0,0 +1,90 @@
+import random
+import torch
+import numpy as np
+from typing import Union, Optional
+
+from tianshou.policy import DQNPolicy
+from tianshou.data import Batch, to_numpy
+
+
+class MADQNPolicy(DQNPolicy):
+    """DQN for multi-agent RL.
+
+    :param torch.nn.Module model: a model following the rules in
+        :class:`~tianshou.policy.BasePolicy`. (s -> logits)
+    :param torch.optim.Optimizer optim: a torch.optim for optimizing the model.
+    :param float discount_factor: in [0, 1].
+    :param int estimation_step: greater than 1, the number of steps to look
+        ahead.
+    :param int target_update_freq: the target network update frequency (``0``
+        if you do not use the target network).
+    :param int agent_id: which agent is the policy playing
+
+    .. seealso::
+
+        Please refer to :class:`~tianshou.policy.BasePolicy` for more detailed
+        explanation.
+    """
+
+    def __init__(self,
+                 model: torch.nn.Module,
+                 optim: torch.optim.Optimizer,
+                 discount_factor: float = 0.99,
+                 estimation_step: int = 1,
+                 target_update_freq: Optional[int] = 0,
```

Comment:
`0` is valid default ?

Reply:
In the current DQN implementation, 0 is reserved for no target network.